src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/Et1VettingController.java

```diff
@@ -0,0 +1,66 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.controllers;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.et.common.model.ccd.CCDCallbackResponse;
+import uk.gov.hmcts.et.common.model.ccd.CCDRequest;
+import uk.gov.hmcts.et.common.model.ccd.CaseDetails;
+import uk.gov.hmcts.ethos.replacement.docmosis.service.Et1VettingService;
+import uk.gov.hmcts.ethos.replacement.docmosis.service.VerifyTokenService;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import static uk.gov.hmcts.ethos.replacement.docmosis.helpers.CallbackRespHelper.getCallbackRespEntityNoErrors;
+
+@Slf4j
+@RestController
+public class Et1VettingController {
+
+    private final VerifyTokenService verifyTokenService;
+    private final Et1VettingService et1VettingService;
+
+    public Et1VettingController(VerifyTokenService verifyTokenService, Et1VettingService et1VettingService) {
```

Comment:
Personally I think using the lombok @RequiredArgsConstructor is a good thing as it enables us to remove boilerplate code (in this case the constructor).